dll/win32/browseui/internettoolbar.cpp

```diff
@@ -715,10 +719,31 @@ HRESULT CInternetToolbar::LockUnlockToolbars(bool locked)
     int                                     bandCount;
     CDockSite                               *dockSite;
     HRESULT                                 hResult;
+    HKEY                                    hKey;
 
     if (locked != fLocked)
     {
+        if (RegCreateKeyExW(HKEY_CURRENT_USER,
```

Comment:
I was hoping commenting on one function would be enough, but any reason why you are not using `SHRegSetUSValueW` here?